polkadot/runtime/parachains/src/coretime/mod.rs

```diff
@@ -247,7 +252,7 @@ fn mk_coretime_call(call: crate::coretime::CoretimeCalls) -> Instruction<()> {
 		origin_kind: OriginKind::Superuser,
 		// Largest call is set_lease with 1526 byte:
 		// Longest call is reserve() with 31_000_000
-		require_weight_at_most: Weight::from_parts(100_000_000, 20_000),
+		require_weight_at_most: Weight::from_parts(110_000_000, 20_000),
```

Comment:
Was it really that tight?

Reply:
Yes. I think 106_000_000 or so was it AFAIR.